run/o2sim_parallel.cxx

```diff
@@ -512,6 +512,46 @@ int main(int argc, char* argv[])
     }
     return r;
   }
+  // check if Kinematics input file is provided
+  LOG(info) << "Kinematics input file: " << conf.getKineInput();
```

Comment:
We have a generator called `GeneratorFromO2Kine`. This may already be doing most of the work... but we can extend it do be able to directly reading from Alien, applying some timeframe offsetting, and performing additional random modification (such as rotations).